Test/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NUnit" version="3.9.0" targetFramework="net45" />
-  <package id="NUnit3TestAdapter" version="3.9.0" targetFramework="net45" />
+  <package id="NUnit" version="3.10.1" targetFramework="net471" />
```

Comment:
should be 461? do you need to get whatever NUnit nuget package uses 461?

Reply:
Hmm, good thoughts. I don't know. The tests do run with this version.